packages/broker/test/integration/storage/StoragePlugin.test.ts

```diff
@@ -0,0 +1,87 @@
+import { MetricsContext } from 'streamr-network'
+import { StoragePlugin } from '../../../src/storage/StoragePlugin'
+import { StorageConfig } from '../../../src/storage/StorageConfig'
+import { createMockStorageConfig } from './MockStorageConfig'
+import { fastPrivateKey } from '../../utils'
+import { StreamPart } from '../../../src/types'
+
+const STREAM_PARTS: StreamPart[] = [ 
+    { id: 'foo', partition: 0 },
+    { id: 'bar', partition: 0 }
+]
+
+const createMockPlugin = (networkNode: any, subscriptionManager: any) => {
+    const brokerConfig: any = {
+        ethereumPrivateKey: fastPrivateKey(),
+        network: {
+            isStorageNode: true
```

Comment:
This should no longer have to be `true`?

Reply:
Fixed